packages/stream-deck-plugin/CLAUDE.md

```
@@ -407,24 +411,24 @@ Key points:
 
-All action source files are in `packages/actions/src/actions/`.
+All action source files are in `packages/actions/src/actions/<action-name>/<action-name>.ts`.
 
 | Pattern | Example |
 |---------|---------|
-| Telemetry-aware icon (single control) | `car-control.ts` (pit-speed-limiter) |
-| Telemetry-aware icon (full action) | `tire-service.ts` |
+| Telemetry-aware icon (single control) | `car-control/car-control.ts` (pit-speed-limiter) |
+| Telemetry-aware icon (full action) | `tire-service/tire-service.ts` |
 
 ### Reference implementations
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)